src/Jank/Scorepads/types.ts

```diff
@@ -0,0 +1,23 @@
+export interface Player {
+  id: string,
+  name: string,
+  picture: string,
+}
+
+export interface Scorepad {
+  id: string,
+  game: string,
+  players: Player[],
+  date: string,
+}
+
+export enum ScorepadActionType {
+  Set,
+  Add,
+  Delete,
+}
+
+export interface ScorepadAction {
+  type: ScorepadActionType,
```

Comment:
It's probably like this for term actions, but we could even be more precise here. You don't need to fix it in this PR, but I think it would be better to name the actions explicitly, like:
```
export interface SetScorepadsAction {
  type: ScorepadActionType.Set,
  payload: Scorepad[],
}

export interface AddScorepadAction {
  type: ScorepadActionType.Add,
  payload: Scorepad,
}

export interface DeleteScorepadAction {
  type: ScorepadActionType.Delete,
  payload: string,
}
```
and then just use a union type for `ScorepadAction` like this
```
export type ScorepadAction = SetScorepadsAction | AddScorepadAction | DeleteScorepadAction;
```
This prevents invalid actions, like `type: Add` with `payload: string` (which would be valid right now).
In addition, we don't need to type cast anything in the reducer, so
```
action.payload as Scorepad[];
```
could just be
```
action.payload;
```
because the compiler knows that `type: Set` always has `Scorepad[]` as its payload. Right now we are cheating the compiler a bit 😬